tests/testthat/test-expand.R

```diff
@@ -126,3 +126,44 @@ test_that("crossing handles list columns", {
   expect_equal(out$x, rep(x, each = 2))
   expect_equal(out$y, rep(y, 2))
 })
+
+test_that("expand_grid can control name_repair", {
+  x <- 1:2
+
+  if (packageVersion("tibble") > "2.99") {
+    expect_error(expand_grid(x, x), class = "vctrs_error_names_must_be_unique")
+  } else {
+    expect_error(expand_grid(x, x), "must not be duplicated")
+  }
+
+  expect_message(
+    out <- expand_grid(x, x, .name_repair = "unique"),
+    "New names:"
+  )
+  expect_named(out, c("x...1", "x...2"))
+
+  out <- expand_grid(x, x, .name_repair = "minimal")
+  expect_named(out, c("x", "x"))
+})
+
+test_that("crossing/nesting/expand respect .name_repair", {
+
+  x <- 1:2
+  expect_named(
```

Comment:
Could you please fit each of these on a single line?

Reply:
Done